code/gamemode/Game.Settings.cs

```diff
@@ -3,11 +3,20 @@ namespace TTTReborn.Settings
     public partial class ServerSettings
     {
         public Categories.Round Round { get; set; } = new Categories.Round();
+        public Categories.AFK AFK { get; set; } = new Categories.AFK();
     }
 
 
     namespace Categories
     {
+        public partial class AFK
+        {
+            /// <value><c>ShouldKickPlayers</c> is if the system should kick the player versus moving them to spectators.</value>
+            public bool ShouldKickPlayers { get; set; } = true;
```

Comment:
I think by default original TTT used to just move players to spectator instead of kicking them, I kinda of agree with sticking with this default. Thoughts?

Reply:
Giving them the option to kick over just moving to spectate is a great idea, but I agree that the default should likely be what people are used to in previous versions.